sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/integration/ReactiveAuditableIT.java

```diff
@@ -28,6 +32,11 @@
 @ContextConfiguration(classes = TestRepositoryConfig.class)
 public class ReactiveAuditableIT {
 
+    @ClassRule
+    public static final IntegrationTestCollectionManager collectionManager = new IntegrationTestCollectionManager();
+
+    @Autowired
+    private CosmosTemplate template;
```

Comment:
Why do we need `CosmosTemplate` instance for reactive auditable integration test ?

Reply:
Because I was being lazy, I'll update the relevant classes to support either CosmosTemplate or ReactiveCosmosTemplate.

Reply:
Thanks, that would be wonderful. I would like to see a clear separation of these two templates, if we can. That would enable us to modify either of them, without affecting the other group.